src/mmw/js/src/draw/views.js

```diff
@@ -67,6 +67,7 @@ function validateShape(polygon) {
         message += Math.floor(area) + ' square km were selected, ';
         message += 'but the maximum supported size is currently ';
         message += MAX_AREA + ' square km.';
+        window.alert(message);
```

Comment:
I do too. Let me do a git history inspection to see if I can pinpoint when we lost it